deploy/requirements.txt

```diff
@@ -22,10 +22,10 @@ django-redis-cache==3.0.0
 drf_dynamic_fields==0.3.0
 drf_yasg==1.20.0
 drf-generators==0.3.0
-label-studio-converter==0.0.26
+label-studio-converter==0.0.27
```

Comment:
let's stick on either pointer to git commit or publish some `.dev` versions to make it work